pxtsim/simlib.ts

```diff
@@ -219,6 +219,9 @@ namespace pxsim {
         export function mute(mute: boolean) {
             _mute = mute;
             stopAll();
+            const ctx = context();
+            if (!mute && ctx && ctx.state === "suspended")
+                ctx.resume();
```

Comment:
why is mute resuming?

Reply:
The resume call has to be made explicitly from a user action, and the context state is initially suspended in iOS. I could bump it to context, but I'd still need an arbitrary call to context or something else in mute here

Reply:
I got confused with what "context" was. LGTM!